src/passes.rs

```diff
@@ -1008,28 +1011,28 @@ impl<T: PassManagerSubType> PassManager<T> {
         unsafe { LLVMAddLoopUnrollAndJamPass(self.pass_manager) }
     }
 
-    #[llvm_versions(8.0..=latest)]
+    #[llvm_versions(8.0..15.0)]
     pub fn add_coroutine_early_pass(&self) {
         use llvm_sys::transforms::coroutines::LLVMAddCoroEarlyPass;
 
         unsafe { LLVMAddCoroEarlyPass(self.pass_manager) }
     }
 
-    #[llvm_versions(8.0..=latest)]
+    #[llvm_versions(8.0..15.0)]
     pub fn add_coroutine_split_pass(&self) {
         use llvm_sys::transforms::coroutines::LLVMAddCoroSplitPass;
 
         unsafe { LLVMAddCoroSplitPass(self.pass_manager) }
     }
 
-    #[llvm_versions(8.0..=latest)]
+    #[llvm_versions(8.0..15.0)]
     pub fn add_coroutine_elide_pass(&self) {
         use llvm_sys::transforms::coroutines::LLVMAddCoroElidePass;
 
         unsafe { LLVMAddCoroElidePass(self.pass_manager) }
     }
 
-    #[llvm_versions(8.0..=latest)]
+    #[llvm_versions(8.0..15.0)]
```

Comment:
Are these getting removed in 16?

Reply:
They are already removed in 15.

Reply:
The old pass manager is getting obsolete in favor of the new one, so they must be removing as much as possible.